ansible/roles/couchdb/tasks/deploy.yml

```diff
@@ -23,6 +23,22 @@
     volume_dir: "{{ instance.volume.fsmount | default( '/mnt/' + group_names|first, true ) }}:/usr/local/var/lib/couchdb"
   when: (block_device is defined) and (block_device in disk_status.stdout)
 
+- name: "generate erlang cookie"
```

Comment:
Thanks for submitting this PR!
Do we need to generate the cookie even if a single couchdb node is deployed?

Reply:
don't need to